hack/ldapsearch/README.md

```diff
@@ -0,0 +1,19 @@
+# ldapsearch.sh
+
```

Comment:
Do we want a `/hack/debugging` dir?
I recall a script for helping debug certificates as well, although that may have had a tanzu specific use case.